guides/v2.1/ui_comp_guide/concepts/ui_comp_data_source.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: default
+group: UI_Components_guide
+subgroup: concepts
+title: The DataSource Component
+menu_title: Providing Data to UI Components
+menu_order: 5
+contributor_name: SwiftOtter Studios
+contributor_link: https://swiftotter.com/
+version: 2.1
+github_link: ui_comp_guide/concepts/ui_comp_data_source.md
+---
+
+
+## {{page.title}}
+{:.no_toc}
+
+
+## Overview
+Magento provides a component that is designed to interact with data in your component. Many of the core UI Components use this DataSource component. Many UI components also require that this component is included. However, there are specific requirements it has in order for it to work correctly. 
+
```

Comment:
Suggested change to "Magento provides the DataSource object, which is designed to interact with data in your UI component."

(This way we are being very specific immediately about what we are talking about in the first sentence, which is the DataSource object, instead of being vague and saying "Magento provides a component...")